### ISSUE TITLE
 Forbes Billionaire Data Analysis

### ISSUE THREAD

**Issue by riyatyag-6** [abhisheks008/ML-Crate#496]
### ML-Crate Repository (Proposing new issue)
:red_circle: **Project Title** : Forbes Billionaire Data Analysis
:red_circle: **Aim** : To perform Exploratory Data Analysis(EDA)
:red_circle: **Dataset** :https://www.kaggle.com/datasets/ravillatejakumar/billionaries-dataset
:red_circle: **Approach**: First Data Cleaning is performed then EDA will be done by using python libraries.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : Riya Tyagi
- GitHub Profile Link : https://github.com/riyatyag-6
- Participant ID (If not, then put NA) :  NA
- Approach for this Project :First Data Cleaning is performed then EDA will be done by using python libraries.
- What is your participant role? (Mention the Open Source Program name. Eg. HRSoC, GSSoC, GSOC etc.):JWOC'24

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by riyatyag-6** [comment]
Kindly Assign this issue to me.
Thanks

**Comment by CodexAbhi** [comment]
name: Abhimanyu Jaiswal
github profile: https://github.com/CodexAbhi
participant id: NA
approach: will try to fill missng values and clean data, check for outliers and try to make data useful for applying models. will do proper documentation along with using graphs and charts to study data.
participant role: IWOC

**Comment by abhisheks008** [comment]
Hi @riyatyag-6 this project is already present in this repository, https://github.com/abhisheks008/ML-Crate/tree/main/Billionares%20Statistics%20Analysis%20Model.

If you wanna enhance this project you can share your approach.

**Comment by hemant933** [comment]
name: Hemant chaudhary
github profile: https://github.com/hemant933
participant id: NA
approach: will try to fill missng values and clean data, check for outliers and try to make data useful for applying models. will do proper documentation along with using graphs and charts to study data.
participant role: IWOC

assign it to me

**Comment by abhisheks008** [comment]
Check out the previous comments.

**Comment by pawaspy** [comment]
Full name : Pawas Pandey
GitHub Profile Link : https://github.com/pawaspy
Participant ID (If not, then put NA) : NA
Approach for this Project :First Data Cleaning is performed then EDA will be done by using python libraries and then model will be developed.
What is your participant role? (Mention the Open Source Program name. Eg. HRSoC, GSSoC, GSOC etc.):IWOC'24

**Comment by abhisheks008** [comment]
Check out the previous comments.

**Comment by pawaspy** [comment]
@abhisheks008 The link you provided in the above comment to the model the person has never used any model. He has just visualized the data and also the dataset has also been changed. It hase new columns and updated the dataset.

**Comment by abhisheks008** [comment]
Do you want to enhance this project https://github.com/abhisheks008/ML-Crate/tree/main/Billionares%20Statistics%20Analysis%20Model?

**Comment by pawaspy** [comment]
@abhisheks008 yes assign me this issue.
Full name : Pawas Pandey
GitHub Profile Link : https://github.com/pawaspy
Participant ID (If not, then put NA) : NA
Approach for this Project :First Data Cleaning is performed then EDA will be done by using python libraries and then I will create model and test the data.
What is your participant role? (Mention the Open Source Program name. Eg. HRSoC, GSSoC, GSOC etc.):JWoC'24

**Comment by abhisheks008** [comment]
Do you want to enhance this project https://github.com/abhisheks008/ML-Crate/tree/main/Billionares%20Statistics%20Analysis%20Model?

**Comment by pawaspy** [comment]
@abhisheks008 Yes. I want to enhance the project. I will add different models in this and more analysis on the data.

**Comment by abhisheks008** [comment]
Assigned under JWOC @pawaspy

**Comment by shivansh-2003** [comment]
Can You Please Assign this issue under SSOC. 2024 Season 3
Shivansh Mahajan
Github:- https://github.com/shivansh-2003
Participation ID:- NA
I will do EDA of the data set by various statistical methods like IQR , Study Distribution OF Feature and Correlation Matrix.
I would do every data cleaning steps like duplication removal and nan value feeding . I would do draw data Analysis by various representing methods like bar , pie and Histogram.
. I am well versed with Data Analysis you can check out my linkedin :-https://www.linkedin.com/in/shivansh-mahajan-13227824a/ and Git repository .
can u assign me with this issue @abhisheks008
Participation Role:- SSOC Season 3

**Comment by abhisheks008** [comment]
Contributions will start from June 1, 2024. Till then please have some patience.

**Comment by Tanishka023** [comment]
Hey @abhisheks008 Sir, please assign this issue to me, I really want to work on this.

**Comment by abhisheks008** [comment]
> Hey @abhisheks008 Sir, please assign this issue to me, I really want to work on this.

Please share your approach in a detailed manner.

**Comment by Tanishka023** [comment]
Full name : Tanishka Bhalla
GitHub Profile Link: https://github.com/Tanishka023
Participant ID (If not, then put NA): NA
Approach for this project: Performing EDA first, followed by applying statistical and machine learning techniques to identify trends, correlations, and factors influencing billionaire wealth accumulation.
What is your participant role? SSoC Season 3

**Comment by abhisheks008** [comment]
> Full name : Tanishka Bhalla GitHub Profile Link: https://github.com/Tanishka023 Participant ID (If not, then put NA): NA Approach for this project: Performing EDA first, followed by applying statistical and machine learning techniques to identify trends, correlations, and factors influencing billionaire wealth accumulation. What is your participant role? SSoC Season 3

What are the machine learning models you are planning to implement?
Mention 5-6 models in your approach.

**Comment by Tanishka023** [comment]
> > Full name : Tanishka Bhalla GitHub Profile Link: https://github.com/Tanishka023 Participant ID (If not, then put NA): NA Approach for this project: Performing EDA first, followed by applying statistical and machine learning techniques to identify trends, correlations, and factors influencing billionaire wealth accumulation. What is your participant role? SSoC Season 3
> 
> What are the machine learning models you are planning to implement?
> Mention 5-6 models in your approach.

SVM, Linear Regression, Random Forests, Decision Trees, NN.

**Comment by abhisheks008** [comment]
> > > Full name : Tanishka Bhalla GitHub Profile Link: https://github.com/Tanishka023 Participant ID (If not, then put NA): NA Approach for this project: Performing EDA first, followed by applying statistical and machine learning techniques to identify trends, correlations, and factors influencing billionaire wealth accumulation. What is your participant role? SSoC Season 3
> > 
> > 
> > What are the machine learning models you are planning to implement?
> > Mention 5-6 models in your approach.
> 
> SVM, Linear Regression, Random Forests, Decision Trees, NN. @abhisheks008

Implement all the above mentioned models for this dataset.

Assigned @Tanishka023
